src/bout++.cxx

```diff
@@ -87,6 +87,7 @@ void bout_signal_handler(int sig);  // Handles signals
 #include <output.hxx>
 
 BoutReal simtime;
+RunMetrics run_data;
```

Comment:
Can this be a member of `BoutMonitor` instead of being a global? You'd need to also add a constructor to `BoutMonitor` and move the `run_data.outputVars` call to there.